### ISSUE TITLE
letters disappear while typing 

### ISSUE THREAD

**Issue by mariusa** [geospoc/v-mapbox-geocoder#60]
When typing in search box, quickly, eg Viscri, the 4th letter is removed as autocomplete results are shown. This is unusable.

**Comment by karol-bujacek** [comment]
I agree with @mariusa, current behavior is really unusable. It seems to me that it is directly related to the `:input.sync='...'` attribute.
